etc/profile-m-z/telegram.profile

```diff
@@ -25,5 +25,21 @@ seccomp
 
 disable-mnt
 private-cache
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+private-etc ca-certificates,crypto-policies,fonts,ld.so.cache,localtime,machine-id,pki,pulse,resolv.conf,ssl
+=======
+private-etc fonts,resolv.conf,ld.so.cache,localtime,ca-certificates,ssl,pki,pulse
+>>>>>>> Create telegram.profile
+=======
+private-etc fonts,resolv.conf,ld.so.cache,localtime,ca-certificates,ssl,pki,pulse
+>>>>>>> Revert "Delete telegram.profile"
+=======
+private-etc fonts,resolv.conf,ld.so.cache,localtime,ca-certificates,ssl,pki,pulse
+>>>>>>> Revert "Delete telegram.profile"
+=======
+private-etc ca-certificates,crypto-policies,fonts,ld.so.cache,localtime,machine-id,pki,pulse,resolv.conf,ssl
+>>>>>>> parent of 17517516... Create telegram.profile
```

Comment:
^

Reply:
@Neo00001 commenting does not help.

Reply:
It seems I have messed up this one. Can you help me out? btw,I'm using github-desktop

Reply:
I have no experience with github-desktop. I found [this](https://stackoverflow.com/questions/37492908/is-there-a-way-to-make-github-desktop-rebase-a-branch-against-master). IDK if this is helpfull.

----

On the commandline it would be.
```
git checkout your_branch
git rebase -i master # where master should be at same HEAD as netblue30:master
```

https://docs.github.com/en/github/using-git/about-git-rebase
https://docs.github.com/en/github/using-git/using-git-rebase-on-the-command-line

Keep in mind that rebasing it a create thing to totally mess up a history (i.e. have a backup or first try in a test copy).

Reply:
`git rebase -i master` - it's showing 
```
etc/profile-m-z/telegram.profile: needs merge
error: cannot rebase: You have unstaged changes.
error: additionally, your index contains uncommitted changes.
error: Please commit or stash them.
```

what shoud I do?

Reply:
Can this be deleted? So that I can re-open a new PR.